src/components/OfficeSections/OfficeSectionsComponent.jsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Header,
+  Table,
+  Pagination,
+  Segment,
+  Divider,
+  Button
+} from 'semantic-ui-react';
+import _ from 'lodash';
+
+import TableRow from '../TableRowComponent';
+import NavBarComponent from '../../_components/NavBarContainer';
+import DropdownComponent from '../../_components/DropdownComponent';
+import LoaderComponent from '../../components/LoaderComponent';
+import rowOptions from '../../_utils/pageRowOptions';
+import ModalComponent from '../common/ModalComponent';
+import ItemsNotFoundComponent from '../common/ItemsNotFoundComponent';
+
+export class OfficeSectionsComponent extends React.Component {
+  state = {
+    activePage: 1,
+    limit: 10
+  };
+
+  componentDidMount() {
+    this.props.loadOfficeSections(this.state.activePage, this.state.limit);
+  }
+
+  handleRowChange = (e, data) => {
```

Comment:
The event is not used at all. If so, then you might want to omit it entirely. Alternatively, this is a good place to call `e.stopPropagation()`. See https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation.
Similar thing is present on handlePaginationChange